lib/github-checks.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+const chalk = require('chalk');
+const execa = require('execa');
+const ghGot = require('gh-got');
+const hostedGitInfo = require('hosted-git-info');
+const pTimeout = require('p-timeout');
+const pWaitFor = require('p-wait-for');
+const terminalLink = require('terminal-link');
+const util = require('./util');
+
+const THIRTY_SECONDS = 1000 * 30;
+const THIRTY_MINUTES = 1000 * 60 * 30;
+
+module.exports = async task => {
+	const hasUnpushedCommits = await execa.stdout('git', ['cherry']) !== '';
+
+	if (hasUnpushedCommits) {
+		await execa('git', ['push']);
+	}
+
+	const latestCommit = execa.stdout('git', ['rev-parse', 'HEAD']);
+	const {title} = task;
+	const pkg = util.readPkg();
+	const {user, project} = hostedGitInfo.fromUrl(pkg.repository.url);
```

Comment:
This could also be in the util file.

Reply:
Moved to `util.js` in 8e25256a3f868753cf6636e41cae46a7f8b1c8b1.